packages/api-catalog-service/README.md

```diff
@@ -0,0 +1,25 @@
+ApiCatalog Microservice
+=================================================
+
+One platform's server-side ApiCatalog GraphQL microservice.
```

Comment:
Add more details to the README, and also change the format according to the new readme template used in all the other microservices.

Reply:
updated